docs/user-documentation/objects.md

```diff
@@ -1,26 +1,65 @@
 
-!!! todo
-    * It’s a node, with maybe one or more medias and files attached (see “Datastreams are Media” section)
-    * The node has metadata in its fields (See "Metadata are Fields" section.)
-    * It has a URI, and a JSON-LD view in Drupal
-    * It goes into Fedora. (See todo make "In Fedora" section)
-    * By Drupal default, all published nodes are public, so see todo make Access Control section.
-    * As a Drupal node, it’s available to Views and gets indexed in Solr.
-    * We suggest using a taxonomy term on the node to control elements of its display or editing, e.g. in Islandora Demo there is a taxonomy term that sets the type: Audio, Binary, Collection, Image, and Video
-    * How do I make an Islandora object? (Make a node, then add a Media)
+```
+userid: 1
+title: "I am an Islandora 8 object"
+created: 1550703004
+changed: 1550703512
+uuid: 02932f2c-e4c2-4b7e-95e1-4aceab78c638
+type: islandora_object
+status: 1
+```
 
+These are Drupal node properties, which all Drupal nodes have (although not all Drupal nodes have a type of "islandora_object", some have a type of "page" or "article" or "some_custom_content_type"). These node properties directly correspond to an Islandora 7.x object's properties:
 
-!!! tip "7.x migration note"
-    #### Where are my Solution Packs?
-    If you are familiar with Islandora 7.x, a solution pack is a bundle representing objects of a certain 'type', that are associated with metadata forms, an ingest workflow that allows specific filetypes, derivative settings based on the uploaded object, an optional content viewer, and the look and feel of an object's page including what metadata to display to the user.
+Islandora 7.x object properties | Islandora 8.x node/object properties
+------------ | -------------
+owner | userid
+dc.title | title
+PID | uuid
+content model | type (and also tags from the Islandora Models vocabulary)
+status | status
+
+In addition to these basic node properties, Islandora 8 objects (like all Drupal nodes) can have fields, which is where most of what we would think of as metadata is stored. The section on [metadata](metadata.md) describes how fields work.
+
+Islandora's 7.x's content models do not exist in Islandora 8. The primary way that Islandora 8 identifies what we think of as a conetent model in Islandora 7.x is using taxonomy terms from the "Islandora Models" vocabulary. To indicate that an Islandora 8 object is an image, video, etc, the user selects a model from a select list:
+
+![Media tab](../assets/object_model_tags.png)
+
+The combination of Drupal content type (e.g., islandora_object) and a value from the "Islandora Models" vocabulary (e.g. "Image") are the equivalent of Islandora 7.x's content models. In Islandora 7.x, an object's content model is immutable, but in Islandora 8, an object's model can be changed from within its node edit form using a select list.
```

Comment:
Might be worth making the distinction that the content type is more like the metadata profile, and the models vocab tells you what type of object you have.  E.g. and Collection is a Collection no matter the content_type/metadata standard that gets used, so long as the node is tagged with the Collection term.

Reply:
Good suggestion, I'll take care of that this evening.

Reply:
OK, added a section "Drupal Content Types as Islandora Metadata Profiles".